### PR TITLE
Fixed Typo Mistakes and Fixed Grammar Mistakes

### PR DESCRIPTION
**Pull request by steffinstanly**
----

- [ ] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Manually tested changed features in running JabRef
- [ ] Screenshots added in PR description (for bigger UI changes)
- [ ] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -130,18 +130,18 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
...
-- We add auto url formatting when user paste link to URL field in entry editor. [koppor#254](https://github.com/koppor/jabref/issues/254)
-- We added a minimal height for the entry editor so that it can no longer be hidden by accident. [#4279](https://github.com/JabRef/jabref/issues/4279)
+- We add auto URL formatting when user paste link to URL field in entry editor. [koppor#254](https://github.com/koppor/jabref/issues/254)
+- We added a minimum height for the entry editor so that it can no longer be hidden by accident. [#4279](https://github.com/JabRef/jabref/issues/4279)
 - We added a new keyboard shortcut so that the entry editor could be closed by <kbd>Ctrl</kbd> + <kbd>E</kbd>. [#4222] (https://github.com/JabRef/jabref/issues/4222)
 - We added an option in the preference dialog box, that allows user to pick the dark or light theme option. [#4130] (https://github.com/JabRef/jabref/issues/4130)
-- We updated updated the Related Articles tab to accept JSON from the new version of the Mr. DLib service
+- We updated the Related Articles tab to accept JSON from the new version of the Mr. DLib service
```
Comment (koppor) [inline CHANGELOG.md]: Shouldn't there be quotes around "Related Articles"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -177,7 +177,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed an issue where the group tree was not updated correctly after an entry was changed. https://github.com/JabRef/jabref/issues/3618
 - We fixed an issue where a right-click in the main table selected a wrong entry. https://github.com/JabRef/jabref/issues/3267
 - We fixed an issue where in rare cases entries where overlayed in the main table. https://github.com/JabRef/jabref/issues/3281
-- We fixed an issue where selecting a group messed up the focus of the main table / entry editor. https://github.com/JabRef/jabref/issues/3367
+- We fixed an issue where selecting a group messed up the focus of the main table/entry editor. https://github.com/JabRef/jabref/issues/3367
```
Comment (koppor) [inline CHANGELOG.md]: "main table" and "entry editor" are two different things in JabRef. Not sure whether the "table/entry" is read together by the ready with your fix. Maybe reword? -- "main table and the entry editor"?

File: CHANGELOG.md (hunk 1 above)
Comment (steffinstanly) [reply CHANGELOG.md]: I guess it's not important

File: CHANGELOG.md (hunk 2 above)
Comment (steffinstanly) [reply CHANGELOG.md]: yes what you suggested would be a batter option

**Comment by koppor** [comment]
Thank you for your PR. We are looking forward to code contributions from your side. There are many open issues in this project.
